### DISCUSSION TITLE
Allow custom S3 endpoint when using docker-compose

### DISCUSSION THREAD

**Discussion by HendrikSchrieber** [developmentseed/titiler#357] [Feature Request]
When using an AWS S3 COS from a different provider a custom endpoint url has to be specified for Rasterio/Gdal to be able to access files from the bucket. Currently, only AWS_SECRET_ACCESS_KEY and AWS_ACCESS_KEY_ID environment variables are passed to Docker in docker-compose.yml. It would be great if AWS_S3_ENDPOINT could be added to docker-compose.yml to be able to use different S3 providers.

**Comment by vincentsarago** [comment]
@Hendrik-LGLN Feel free to start a PR to add this 👍
